Assets/Scripts/Nodes/BuddyNodes.cs

```diff
@@ -47,6 +49,7 @@ public override NodeStatus Tick()
 		offset.z = offset.y;
 		offset.y = 0;
 		baseInfo.destination = transform.position + offset;
+		controller.moveDirection = Vector3.zero;
```

Comment:
It shouldn't be there. Since the node isn't related to movement, it shouldn't be affecting movement. If we want to be sure that the buddy stops moving when the movement node ends, the movement node should be doing taking care of that.